packages/design-system-react/src/components/text/Text.stories.tsx

```diff
@@ -167,7 +168,13 @@ export const AsChild: Story = {
         <span>Text rendered as span</span>
       </Text>
       <Text asChild className="block">
-        <button type="button" onClick={action('button-clicked')}>
+        <button
+          type="button"
+          onClick={() => {
+            // eslint-disable-next-line no-alert, no-restricted-globals
+            alert('button-clicked');
```

Comment:
This was previously using storybook-actions but once we add it as a dev dependency it clashes with the @storybook/addon-actions version we can fix it in a subsequent PR